.env.development

```diff
@@ -3,7 +3,7 @@ LMS_BASE_URL='http://localhost:18000'
 ECOMMERCE_BASE_URL='http://localhost:18130'
 DISCOVERY_API_BASE_URL='http://localhost:18381'
 ENTERPRISE_CATALOG_API_BASE_URL='http://localhost:18160'
-ORDER_HISTORY_URL='http://localhost:1996/orders'
+LICENSE_MANAGER_URL='http://localhost:18170'
```

Comment:
Add LICENSE_MANAGER_URL to edx-internal, along with CORS_ORIGIN_WHITELIST: https://github.com/edx/edx-internal/pull/2219